uspace/app/bdsh/cmds/modules/alias/alias.c

```diff
@@ -73,14 +73,37 @@ static void set_alias(const char *name, const char *value)
 		alias_t *data = odict_get_instance(alias_link, alias_t, odict);
 		free(data->value);
```

Comment:
It would be better to only remove the previous value if the allocation succeeds. I.e. make no change on failure.

Reply:
Good call